docs/pages/installation/eks.md

```diff
@@ -9,106 +9,73 @@ list: include
-Once configured, proceed with using Meshery:
-
-`mesheryctl system start`
-
-#### Manual Configuration (Optional)
-
-If the [Automatic Configuration](#automatic-configuration-recommended) procedure fails or you would like to manually prepare your kubeconfig file to provide Meshery with the necessary access to your managed Kubernetes instance, perform the following actions:
-
-1. Create a *ServiceAccount* with *cluster-admin* role
-    
-    ```$ kubectl create serviceaccount meshery```
-
-1. Adding/Binding *cluster-admin* role to new service account meshery
-    
-    ```$ kubectl create clusterrolebinding meshery-binding --clusterrole=cluster-admin\--serviceaccount=default:meshery```
 
-1. Get secret name from *ServiceAccount*.
-    
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl get secrets
-
-    NAME                           TYPE                                  DATA   AGE
-    default-token-fnfjp            kubernetes.io/service-account-token   3      95d
-    meshery-token-5z9xj               kubernetes.io/service-account-token   3      66m
-    </div></div>
-    </pre>
-
-    _Note: Here the secret name is meshery-token-5z9xj_
-1. Get secret/token:
-
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl describe secret  sa-1-token-5z9xj
-    Name:         meshery-token-5z9xj
-    Namespace:    default
-    Labels:       <none>
-    Annotations:  kubernetes.io/service-account.name: meshery
-                  kubernetes.io/service-account.uid: 397XXX-XXX-XXXX-XXXXX-XXXXX
-
-    Type:  kubernetes.io/service-account-token
-
-    Data
-    ====
-    ca.crt:     1025 bytes
-    namespace:  7 bytes
-    token:      XXXhbGciOiJSUXXXX     </div></div></pre>
-
-1. Generate new kubeconfig yaml file to use as input to Meshery.
-1. Set config Credential using above generate `token`.
-    
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl config set-credentials meshery --token=XXXXX
-
-    o/p:User "meshery" set.
-    </div></div>
-    </pre>
-
-1. Set current context to our new service account `meshery`
-    
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl config set-context --current --user=meshery
-
-    o/p:
-    Context "aws" modified.
-    </div></div>
-    </pre>
-
-1. Generate kubeconfig yaml file to use as input to Meshery.
-    
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl config view --minify --flatten >  config_aws_eks.yaml
-    </div></div>
-    </pre>
-
-Meshery should now be connected with your managed Kubernetes instance. Take a look at the [Meshery guides]({{ site.baseurl }}/guides) for advanced usage tips.-->
+{% include installation_prerequisites.html %}
 
-*Coming soon*
\ No newline at end of file
+## To set up and run Meshery on EKS:
+
```

Comment:
@leecalcote @navendu-pottekkat @alphaX86 would it be a nice UX to have separate instructions based on if the user wants to deploy Meshery in Docker or in k8s?
From my POV and with some hands-on it would be.

Reply:
The only requirement when installing meshery "on" EKS using `mesheryctl system start` is that your kubeconfig is pointing to the EKS. An EKS user should know this because they would need this to use even simple kubectl command. However, it would be still nice to mention that this can be done using the command,
```
aws eks update-kubeconfig \
  --region <region-name> \
  --name <cluster-name>
```